src/email/interfaces/aws-email.config.interface.ts

```diff
@@ -0,0 +1,5 @@
+export interface IAwsEmailConfig {
```

Comment:
Rename this interface as AwsEmailConfig since it's the recommended naming convention in NestJs. I will fix IPlugin naming also.